### PR TITLE
remove L1Penalty.clearState

### PR DESCRIPTION
**Pull request by clement-masson**
Calling L1Penalty's clearState resulted on an error.
The loss being a scalar, it has no .set method.

```
local loss = m*input:norm(1) 
self.loss = loss  
```
The default Module.clearState should be ok, so I simply removed the L1Penalty's override.

### CODE REVIEW

**Comment by soumith** [comment]
Thanks!

**Comment by clement-masson** [comment]
you're welcome :)
